content/pages/team.md

```diff
@@ -5,7 +5,7 @@ license: https://www.apache.org/licenses/LICENSE-2.0
 The ASF Tooling team, known as **Tooling**, provides tooling development for select ASF Initiatives.
 
 ## About us
-Tooling has a core team made up of a number of paid contractors, whom are senior software engineers; and volunteers. The core team collaborates to develop specific software, work with and contribute to Infrs. The volunteers bring their skills to bear on their areas of interest, reducing the load on the core team.
+Tooling has a core team made up of a number of paid contractors, whom are senior software engineers; and volunteers. The core team collaborates to develop specific software, work with and contribute to Infras. The volunteers bring their skills to bear on their areas of interest, reducing the load on the core team.
 
 ## How you can help
-Volunteers are the driving force of the ASF. The Tooling team is looking for people willing to help out at whatever effort level suits. If you're interested, visit our [volunteer information](volunteer.html) page.
+Volunteers are the driving force of the ASF. The Tooling team is looking for people willing to help out at whatever effort level suits. If you're interested, visit our [volunteer information](volunteer.md) page.
```

Comment:
Will this be rewritten automatically to .html on [the relevant page](https://tooling.apache.org/team.html)?